classes/Compatibility/backword-compatibility.php

```diff
@@ -23,3 +23,8 @@ class Throwable extends Exception
         }
     }
 }
+
+try {
```

Comment:
Really useful ?

Reply:
No, this code is absolutely unuseful.
I removed it